File: contoh/test_script.py
#!/usr/bin/env python3
"""
Script contoh untuk testing obfuscator.
Ini akan menampilkan beberapa output untuk membuktikan kode berjalan.
"""

import os
import sys
from datetime import datetime

def greeting():
    print("=" * 50)
    print("  SCRIPT INI BERHASIL DIJALANKAN!")
    print("=" * 50)
    print()

def show_info():
    print(f"Waktu: {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}")
    print(f"Python: {sys.version.split()[0]}")
    print(f"Platform: {sys.platform}")
    print(f"Working dir: {os.getcwd()}")
    print()

def calculate():
    print("Test perhitungan:")
    for i in range(1, 6):
        result = i ** 2
        print(f"  {i}^2 = {result}")
    print()

def secret_message():
    message = "Ini adalah pesan rahasia yang ter-obfuscate!"
    print(f"Pesan: {message}")
    print()

def main():
    greeting()
    show_info()
    calculate()
    secret_message()
    print("Script selesai dengan sukses!")

if __name__ == "__main__":
    main()

File: contoh/test_script.js
// Script contoh untuk testing obfuscator
// Ini akan menampilkan beberapa output untuk membuktikan kode berjalan

function greeting() {
    console.log("=".repeat(50));
    console.log("  SCRIPT INI BERHASIL DIJALANKAN!");
    console.log("=".repeat(50));
    console.log();
}

function showInfo() {
    console.log(`Waktu: ${new Date().toISOString()}`);
    console.log(`Node.js: ${process.version}`);
    console.log(`Platform: ${process.platform}`);
    console.log(`Working dir: ${process.cwd()}`);
    console.log();
}

function calculate() {
    console.log("Test perhitungan:");
    for (let i = 1; i <= 5; i++) {
        const result = i ** 2;
        console.log(`  ${i}^2 = ${result}`);
    }
    console.log();
}

function secretMessage() {
    const message = "Ini adalah pesan rahasia yang ter-obfuscate!";
    console.log(`Pesan: ${message}`);
    console.log();
}

function main() {
    greeting();
    showInfo();
    calculate();
    secretMessage();
    console.log("Script selesai dengan sukses!");
}

main();
